### PR TITLE
Sonatype prepare

### PR DESCRIPTION
**Pull request by douglm**
Does this seem correct?

I had a lot of trouble figuring out what intellij was doing with git

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,34 +1,55 @@
...
-		<groupId>org.mnode</groupId>
-		<version>1.16</version>
-	</parent>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.mnode.ical4j</groupId>
+	<groupId>org.bedework</groupId>
 	<artifactId>ical4j-vcard</artifactId>
```
Comment (nblair) [inline pom.xml]: You should change the artifactId here to match the repository name (bw-ical4j-vcard).

File: pom.xml [hunk 2]
```diff
@@ -1,34 +1,55 @@
...
 	<name>iCal4j VCard</name>
-	<version>0.9.6-BW-SNAPSHOT</version>
-	<description>Support for the vCard specification</description>
-	<url>http://ical4j.sourceforge.net</url>
-	<issueManagement>
-		<system>SourceForge.net</system>
-		<url>https://sourceforge.net/tracker/?group_id=107024</url>
-	</issueManagement>
+	<version>1.0.0-SNAPSHOT</version>
+	<description>A fork of ical4j-vcard with customizations foir bedework</description>
```
Comment (nblair) [inline pom.xml]: Typo (foir).

File: pom.xml [hunk 3]
```diff
@@ -154,9 +175,9 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>org.mnode.ical4j</groupId>
-			<artifactId>ical4j</artifactId>
-			<version>1.0.4</version>
+			<groupId>org.bedework</groupId>
+			<artifactId>bw-ical4j-cl</artifactId>
+			<version>1.1.2-SNAPSHOT</version>
```
Comment (nblair) [inline pom.xml]: Since we're working on releasing, don't depend on -SNAPSHOT versions. Change this to the published 1.1.1.

**Comment by douglm** [comment]
This doesn't seem quite right to me - not the updates - but I still can't seem to create a pull request from intellij.

**Comment by nblair** [comment]
Looks good :+1:

**Comment by douglm** [comment]
I guess the build process at cloudbees is partly out of action now we've 
started this process. ical4j fails

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.7:deploy (default-deploy) on project bw-ical4j-cl: Failed to deploy artifacts: Could not transfer artifact org.bedework:bw-ical4j-cl:jar:1.1.2-20151016.092526-1 from/to ossrh (https://oss.sonatype.org/content/repositories/snapshots): Failed to transfer file: https://oss.sonatype.org/content/repositories/snapshots/org/bedework/bw-ical4j-cl/1.1.2-SNAPSHOT/bw-ical4j-cl-1.1.2-20151016.092526-1.jar. Return code is: 401, ReasonPhrase: Unauthorized. -> [Help 1]
> [ERROR]

which I guess is only to be expected.

It does mean that it won't automatically build bw-util because the 
upstream dependency is failing.

However, I managed to trigger off a build manually and it worked - again 
not surprising because bw-util still depends on the earlier version if 
ical4j.

I mention all this because I'm having to apply some fixes to the bw-util 
classes and the notifier and get them deployed. As we work through this 
process I need to make sure I don't end up completely wedged

**Comment by nblair** [comment]
Version 1.1.2-SNAPSHOT is not referenced in the pom here, 1.1.1 is. The problem you are referencing is not related to this branch.

I think you've got the basics in place, so this PR should be ok to merge.

**Pull request merged**